### ISSUE TITLE
Missile range calculation is wrong

### ISSUE THREAD

**Issue by IrateRedKite** [better-modernized-combat/bmod-client#181]
Good example currently visible on the Skipper Light Torpedo launcher. The calculation doesn't account for the short motor time and should probably be revised/adjusted for a more accurate range stat:

```cpp
GetStatLine(flags, &len, 471624, f0fmt, 1759, (gun->fMuzzleVelocity * motor->fLifetime + (motor->fAccel / 2) * std::pow(motor->fLifetime, 2)) + accelerationCap * topSpeed);
```

**Comment by IrateRedKite** [comment]
471615 on line 204 here needs to be swapped as well, as it's displaying an incorrect string and should be 'Detonation Distance' rather than 'Seeker Range'

**Comment by IrateRedKite** [comment]
Completed with b66500d41c599a454644ae2f7b71147adff5399b and 8505a12e0eb0f0cf69c9ab1b914acd686d3897e0

**Comment by IrateRedKite** [comment]
Merged with 3264865c02c424f1c90dda17091256c8da16b853
